api/v1beta1/flowcollector_types.go

```diff
@@ -146,6 +146,19 @@ type FlowCollectorIPFIX struct {
 	OVNKubernetes OVNKubernetesConfig `json:"ovnKubernetes,omitempty" mapstructure:"-"`
 }
 
+// Agent feature, can be one of:<br>
+// - `PKT_DROP`, to track packet drops.<br>
+// - `DNS_TRACKING`, to track specific information on DNS traffic.<br>
+// - `FLOW_RTT`, to track L4 latency. <i>Unsupported (*)</i><br>
```

Comment:
```suggestion
// - `FLOW_RTT`, to track L4 latency. <i>Unsupported (*)</i><br>
```

Should we add a mention that this is only for TCP ?
https://github.com/netobserv/network-observability-console-plugin/pull/365#issuecomment-1693372744